ocaml-lsp-server/src/custom_requests/req_dump_trace.ml

```diff
@@ -0,0 +1,15 @@
+open Import
+
+let meth = "ocamllsp/dumpTrace"
+
+let on_request ~params:_ state =
```

Comment:
How do you use this request? Do you paste the result from the response somewhere and then open chrome?

Reply:
I created a [PR](https://github.com/ocamllabs/vscode-ocaml-platform/pull/588) in vscode-ocaml-platform, which implements a command `Dump Trace` that can be called from the command palette. You should then be able to get the response from output pane either for ocaml-lsp or vscode-ocaml-platform -- both should print that.

Aside: I would generally welcome suggestions on when to enable tracing, how to set N, any suggestions on making tracing nicer...

Reply:
I see. If there's time, I'd experiment with doing it in an editor agnostic way. E.g. send a request to the editor to open a  temp file with the tracing info.